<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_device_watch</class>
 <widget class="QDialog" name="add_device_watch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="OK_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background: rgb(45, 81, 135);
border-radius: 12px;</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
color: rgb(0, 85, 127);</string>
   </property>
   <property name="text">
    <string>ADD (SMARTWATCH)</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>231</width>
     <height>415</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Brand:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="des_lineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Price:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="capacity"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Bluetooth:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="bluetooth_lineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Display:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Capacity:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Srceen size:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>Wifi:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="case_size"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Case size:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="wifi_lineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="brand_lineEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="model_lineEdit"/>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="screen_size"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="price_lineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Quantity:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="display_lineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="quantity_lineEdit"/>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Brightness:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="brightness_lineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>531</width>
     <height>333</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="1">
     <widget class="QLineEdit" name="weight_lineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Dimension:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="water_resistance"/>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="battery_lineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Weight:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>Battery:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QCheckBox" name="has_sim">
      <property name="text">
       <string>has sim</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="dimension2_lineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="dimension1_lineEdit"/>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="has_mic">
      <property name="text">
       <string>has microphone</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Water resistance:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="has_speaker">
      <property name="text">
       <string>has speaker</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="dimension3_lineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="has_blood_O2_sensor">
      <property name="text">
       <string>has Blood O2 sensor</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="has_gps">
      <property name="text">
       <string>has gps</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="has_heartrate_monitering">
      <property name="text">
       <string>heartrate monitoring</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>meter</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>mAh</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
